about.html

```diff
@@ -0,0 +1,127 @@
+        </p>
+        <div class="logo-inside">
+            <img class="about-into-view" src="images/my-logo.png" alt="logo">
+        </div>
+    </section>
+    <section class="third-about-section">
+        <h4 class="about-second-header">Beyond AE Global Summits<hr class="horizontal-line"></h4>
+        <p class="about-second-header-text">
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. In eum aliquam, harum aperiam exercitationem accusantium assumenda blanditiis, quis accusamus ad, ex a eveniet similique rem dolores voluptatem suscipit sapiente consectetur?
+        </p>
+        <div class="desk-about-sum-display">
+            <div class="bg-div-1 item-sum-1">
+                <h2 class="inside-text-date">2020</h2>
+                <p class="sum-loc">AE Global Summit 2023 in Nairobi</p>
+            </div>
+            <div class="bg-div-1 item-sum-1">
+                <h2 class="inside-text-date">2021</h2>
+                <p class="sum-loc">AE Global Summit 2023 in Kampala</p>
+            </div>
+        </div>
+        <div class="bg-div-1 about-sum-ind">
+            <h2 class="inside-text-date">2020</h2>
+            <p class="sum-loc">AE Global Summit 2023 in Nairobi</p>
+        </div>
+        <div class="bg-div-1 about-sum-ind">
+            <h2 class="inside-text-date">2021</h2>
+            <p class="sum-loc">AE Global Summit 2023 in Kampala</p>
+        </div>
+    </section>
+    <section class="about-footer">
+        <div class="partners-about">
+            <h3 class="about-partners">Partner<hr class="horizontal-line"></h3>
+            <div class="about-sponsor-logos">
+                <img class="about-sponsor-logo" src="images/spon-1.svg" alt="sponsor-logo">
+                <img class="about-sponsor-logo"  src="images/spon-6.svg" alt="sponsor-logo">
+                <img class="about-sponsor-logo"  src="images/spon-3.svg" alt="sponsor-logo"><br>
+                
+            </div>
+            <div class="about-sponsor-logos-2">
+                <img class="about-sponsor-logo"  src="images/spon-4.svg" alt="sponsor-logo">
+                <img class="about-sponsor-logo"  src="images/spon-5.svg" alt="sponsor-logo">
+            </div>
+
+        </div>
+        <div class="foot-desk">
+            <div class="main-about-footer">
+                <img class="about-footer-image" src="images/my-logo.png" alt="summit-logo">
+                <div class="about-footer-info">
+                    <h4>2023 Agricultural Enthusiast Nigeria. All Rights Reserved</h4>
+                    <p class="foot-section">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus repellendus, quasi esse nostrum amet accusamus, distinctio reprehenderit quam maiores quae eos exercitationem cum voluptate eligendi unde, quas numquam illum asperiores.</p>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Please kindly put the footer into the `<footer>` tag, and also change the color of your text to `#d3d3d3` to be more visible and readable 😁 
- [ ] Also change the color of the `Partner` title for mobile, it's also dark and is not visible 🙃 

Your design

<img width="984" alt="Screenshot 2023-04-20 at 20 40 20" src="https://user-images.githubusercontent.com/74501523/233445972-921f054a-4491-4442-a3cb-fa7a717af293.png">

Original design

<img width="872" alt="Screenshot 2023-04-20 at 20 40 55" src="https://user-images.githubusercontent.com/74501523/233446026-5532e654-b06c-4a70-b4fa-a66f5fddec62.png">